### PR TITLE
fix #37784 mydirtyhobby.to

### LINKED ISSUE #37784: mydirtyhobby.to
### Issue URL (Incorrect Blocking)
[NSFW][https://mydirtyhobby.to/?lang=ge](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fmydirtyhobby.to%2F%3Flang%3Dge)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 3.0.2
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | None
Filters: | AdGuard Mobile Ads,<br>AdGuard Safari,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard German

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/37784

### CODE REVIEW

File: GermanFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -11,6 +11,8 @@
 !###############
 ! https://github.com/AdguardTeam/AdguardFilters/issues/37287
 @@||st.wetteronline.de/dr/*/css/p_*.css
+! https://github.com/AdguardTeam/AdguardFilters/issues/37784
+@@||mydirtyhobby.to^
```
Comment (Alex-302) [inline GermanFilter/sections/whitelist.txt]: which rule breaks the site?

Comment (zloyden) [reply GermanFilter/sections/whitelist.txt]: That rule `/mydirtyhobby.$domain=~mydirtyhobby.com|~mydirtyhobby.de`. I send report to EasyList team and temporarily added exception.

File: GermanFilter/sections/whitelist.txt [hunk 2]
```diff
@@ -2,6 +2,8 @@
 ! White list. Fixing filtration errors (false-positive)
 !
 !###TEMPORARY###
+! https://github.com/AdguardTeam/AdguardFilters/issues/37784
+@@||mydirtyhobby.to^
```
Comment (Alex-302) [inline GermanFilter/sections/whitelist.txt]: add hint and `$domain=mydirtyhobby.to`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: GermanFilter/sections/whitelist.txt (hunk 2 above)
Comment (Alex-302) [reply GermanFilter/sections/whitelist.txt]: `$domain=mydirtyhobby.to`

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**